src/stream/src/executor/managed_state/join/mod.rs

```diff
@@ -356,7 +441,9 @@ impl<K: HashKey, S: StateStore> JoinHashMap<K, S> {
         }
 
         // If no cache maintained, only update the flush buffer.
-        self.state_table.insert(value.into_row())?;
+        let (row, degree) = value.into_table_rows(&self.state.order_key_indices);
+        self.state.table.insert(row)?;
+        self.degree_state.table.insert(degree)?;
```

Comment:
Do we still need to construct `degree` row and insert it to the state table if `need_degree` is false?